trk/src/Screens/TabScreens/GymAnalytics/GymDaily/Backend/analyticsCalculations.js

```diff
@@ -32,6 +32,10 @@ export const fetchTapsAndCalculateAscents = async (yourClimbs) => {
   let climbFrequencyMap = {};
   let timeFrequencyMap = {};
 
+  for (const climb of yourClimbs) {
+    climbFrequencyMap[climb.id] = { count: 0, name: climb.name, id: climb.id, grade: climb.grade };
+  }
+
   for (const climb of yourClimbs) {
```

Comment:
by initializing  tap count to 0 for each climb, a non-tapped climb is still considered in the calculation for 'least completed route'